pkg/cloudprovider/providers/openstack/openstack.go

```diff
@@ -106,6 +108,7 @@ type Config struct {
 		ApiKey     string `gcfg:"api-key"`
 		TenantId   string `gcfg:"tenant-id"`
 		TenantName string `gcfg:"tenant-name"`
+		TrustId    string `gcfg:"trust-id"`
```

Comment:
so how does this "avoid passing the user credentials in clear inside the config file"? it looks like we still have to have those